player_test.py

```diff
@@ -145,5 +146,55 @@ def test_same_hand_gives_different_scores_for_different_anthill_configurations(s
     self.assertEqual(mario.score, expected_score_a)
     self.assertEqual(luigi.score, expected_score_b)
 
+class ChooseAntToMoveTest(unittest.TestCase):
```

Comment:
The name of this class is consistent to the name of the other test clases you've been using but I've learned this week that this is not a good test name. I suggest you still merge this PR as is but we discuss this in person.